src/js/components/Ballot/PositionItem.jsx

```diff
@@ -17,8 +18,30 @@ export default class PositionItem extends Component {
     var dateText = moment(dateStr).startOf("day").fromNow();
     var speaker_we_vote_id_link = "/voterguide/" + position.speaker_we_vote_id;
 
+    var image_placeholder = "";
+    if (this.props.speaker_type == "O") {
+        image_placeholder = <i className="icon-org-lg icon-icon-org-placeholder-6-2 icon-org-resting-color"></i>
+    } else if (this.props.speaker_type == "V") {
+        image_placeholder = <i className="icon-org-lg icon-icon-person-placeholder-6-1 icon-org-resting-color"></i>
+    }
+
+    var position_description = <span></span>;
+    if (position.vote_smart_rating) {
+        var position_description = <p className="">
```

Comment:
This will get hoisted because there is no block scope unless you use `let`. `var` will get hoisted outside of the block.